pixl_dcmd/src/pixl_dcmd/_database.py

```diff
@@ -53,12 +54,13 @@ def add_hashed_identifier_and_save_to_db(
         return updated_image
 
 
-def query_db(mrn: str, accession_number: str) -> Image:
+def query_db(project_slug: str, mrn: str, accession_number: str) -> Image:
     PixlSession = sessionmaker(engine)
     with PixlSession() as pixl_session, pixl_session.begin():
         existing_image: Image = (
             pixl_session.query(Image)
             .filter(
+                Image.project_name == project_slug,
```

Comment:
then this would also query through the `Extract` table to make sure its the right project